transport/http/peer.go

```diff
@@ -98,7 +129,6 @@ func (p *httpPeer) MaintainConn() {
 	// Attempt to retain an open connection to each peer so long as it is
 	// retained.
 	for {
-		p.Peer.SetStatus(peer.Connecting)
```

Comment:
is it safe to remove this? seems like we're losing track of this state, which might be useful for debugging.